evil-collection.el

```diff
@@ -167,5 +168,8 @@ instance:
           (require (intern (concat "evil-collection-" (symbol-name m))))
           (funcall (intern (concat "evil-collection-" (symbol-name m) "-setup"))))))))
 
+(require 'evil-collection-buffer-menu)
+(funcall 'evil-collection-buffer-menu-setup)
```

Comment:
We shouldn't be doing this here. This item should go in as part of the original mode list.

Reply:
It did not work for some reason

Reply:
You can just add a print statement in the setup function to be called to see if it's being triggered.

Reply:
I used the code
```elisp
(dolist (mode evil-collection-mode-list)
  (let ((m mode)
	(reqs (list mode)))
    (when (listp mode)
      (setq m (car mode)
	    reqs (cdr mode)))
    (dolist (req reqs)
      (with-eval-after-load req
	(print (intern (concat "evil-collection-" (symbol-name m))))
	))))
```
The result in the *Messages* buffer was
```
evil-collection-bookmark
evil-collection-comint
evil-collection-company
evil-collection-compile
evil-collection-custom
evil-collection-debug
evil-collection-diff-mode
evil-collection-dired
evil-collection-elisp-mode
evil-collection-help
evil-collection-image
evil-collection-info
evil-collection-ivy
evil-collection-occur
evil-collection-package-menu
evil-collection-xref
nil
```
A lot of things are missing, in particular buffer-mode. I don't know what to make of it. I also put a print in the function `evil-collection-buffer-mode` and It did not activate, the function is not loaded at all.

Reply:
Try using this file -> https://github.com/jojojames/evil-collection/commit/602d24f07dad7fefe72293bda63a400842b04f62

It's because buff-menu is not provided.